src/Tasks/Resources/Strings.resx

```diff
@@ -2160,7 +2160,11 @@
     <value>MSB3645: .NET Framework v3.5 Service Pack 1 was not found. In order to target "{0}", .NET Framework v3.5 Service Pack 1 or later must be installed.</value>
     <comment>{StrBegin="MSB3645: "}</comment>
   </data>
-
+  <data name="GetReferenceAssemblyPaths.OutOfDateSDK">
+    <value>MSB3961: The reference assemblies for "{0}" were not found. You might be using an older SDK to target .NET 5.0 or higher. Update Visual Studio and/or your .NET SDK.</value>
```

Comment:
[3961 is also in use.](https://github.com/dotnet/msbuild/blob/master/src/Tasks/Resources/Strings.resx#L2840-L2843)

Updating VS right now would mean updating to 16.7. They would have to switch to a preview version, right?

Reply:
This will ship after or at best concurrently with 16.8.

Reply:
Also, this is specifically for customers who are using 16.7 and building a project that targets 5.0.  They are specifically not using 16.8 (preview or otherwise).  They got the project from a coworker or retargeted themselves, etc.  We already have customers reporting this who installed 5.0 previews on top of 16.7 and tried to make it work.

Reply:
@KathleenDollard to review the string.

Reply:
Just wanted to note that you can hit this with 16.8 if you `global.json` yourself down to a 3.1 or lower SDK.